scenario/run_test.go

```diff
@@ -6,37 +6,17 @@ package scenario_test
 
 import (
 	"context"
-	"fmt"
 	"os"
 	"path/filepath"
 	"testing"
 
 	gdtcontext "github.com/gdt-dev/gdt/context"
-	"github.com/gdt-dev/gdt/debug"
 	gdterrors "github.com/gdt-dev/gdt/errors"
-	"github.com/gdt-dev/gdt/result"
 	"github.com/gdt-dev/gdt/scenario"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func (s *fooSpec) Eval(ctx context.Context, t *testing.T) *result.Result {
```

Comment:
note to @a-hilaly I just moved this into the `scenario/stub_plugins_test.go` file along with all the other stub plugin and spec definitions used in unit testing.